GEOSmetforce_GridComp/LDAS_Forcing.F90

```diff
@@ -6646,6 +6704,7 @@ program ut_parse_G5DAS_met_tag
   met_tag_in_vec = (/                              &
        'gcmexpname' ,                          &
        'gcmexpname__Nx+-' ,                    &
+       'gcmexpname__bkg' ,                     &
        'gcmexpname__precCORRPREC',             &
        'gcmexpname__precCORRPREC__Nx+-' ,            &
        'gcmexpname__Nx+-__precCORRPREC' ,            &
```

Comment:
we  may need the option to run bkg **with** precip corr, as in:   `'gcmexpname__bkg__precCORRPREC'` and/or `'gcmexpname__precCORRPREC__Nx+-'`  (not sure whether both options for ordering the substrings are needed (or what the difference may be)
requires more thought

Reply:
Good point.  The forcing in the LADAS is provided by the AGCM surface met forcing outputs.  As long as the corrected precipitation is handled correctly in the AGCM during the Predictor segment, the bkglfo files will already include the corrected precipitation as desired.  So there should **not** be a need to specify a MET_TAG with the corrected precipitation option when LDAS is running in the coupled atm/land DAS. 
We do need to make sure the corrected precipitation works as intended during the **Predictor** segment.  This should be the case, because we wouldn't want the ADAS bkg to have been created with model-generated precipitation.  But I'm not sure if that's ever been verified.